build/configs/artik05x/artik05x_usbrule.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+###########################################################################
+#
+# Copyright 2019 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+#
+# File   : artik05x_usbrule.sh
+# Description : usb rule script for ARTIK 05X
+
+board="artik"
+idVendor="0403"
+idProduct="6010"
```

Comment:
Except those values, there is no difference between each board. How about making a common file and calling it with arguments?

Reply:
modified